tests/lustre/typechecking/shouldPass/const_03.lus

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
This test case is for typing checking connected components in a group, which I will be implementing in the next iteration. 
Do you want to keep this test or push it to next iteration?

Reply:
This test is here to remind me to ask you why the placeholder function for detecting connected components is located after the code that handles constant definitions since it needs the function too.

Reply:
The function which handles constant definitions will split into context extraction and then type checking when the function `scc` for connected components will be implemented. :)

Reply:
Ok. Push it to next iteration.